jujugui/static/gui/src/app/components/deployment/choose-cloud/choose-cloud.js

```diff
@@ -172,74 +180,32 @@ YUI.add('deployment-choose-cloud', function() {
         </div>);
     },
 
-    /**
-      Generate the list of services.
-
-      @method _generateServices
-    */
-    _generateServices: function() {
-      var services = this.props.services;
-      if (services.length === 0) {
-        return;
-      }
-      var components = [];
-      services.forEach((service, i) => {
-        var className = classNames(
-          'two-col',
-          {'last-col': i % 3 === 1}
-        );
-        var name = service.get('name');
-        components.push(
-          <li className={className}
-            key={service.get('id')}>
-            <img alt={name}
-              className="deployment-choose-cloud__services-icon"
-              src={service.get('icon')} />
-            {name}
-          </li>);
-      });
-      return (
-        <ul className="deployment-choose-cloud__services twelve-col">
-          {components}
-        </ul>);
-    },
-
     render: function() {
-      var username = this.props.user && this.props.user.usernameDisplay;
-      var title = `Welcome back, ${username}`;
-      var pluralize = this.props.pluralize;
-      var changeCounts = this.props.changeCounts;
-      var serviceCount = changeCounts['_deploy'] || 0;
-      var machineCount = changeCounts['_addMachines'] || 0;
+      var title = this.state.credentials.length === 0 ?
+        'Choose cloud' : 'Choose cloud or saved credential';
       return (
         <div className="deployment-panel__child">
           <juju.components.DeploymentPanelContent
             title={title}>
-            <div className="six-col">
-              <h3 className="deployment-panel__section-title">
-                Deployment summary&nbsp;
-                <span className="deployment-panel__section-title-count">
-                  ({serviceCount} {pluralize('service', serviceCount)},&nbsp;
-                  {machineCount} {pluralize('machine', machineCount)})
-                </span>
-              </h3>
-              {this._generateServices()}
-            </div>
-            <div className="six-col last-col">
-              <h3 className="deployment-panel__section-title">
-                Unplaced units
-              </h3>
-              <div className="deployment-panel__box">
-              </div>
-            </div>
             {this._generateCredentials()}
             {this._generateOnboarding()}
-            <h3 className="deployment-panel__section-title twelve-col">
-              Public clouds
-            </h3>
             <ul className="deployment-choose-cloud__list twelve-col">
               {this._generateOptions()}
             </ul>
+            <div className="deployment-choose-cloud__download twelve-col">
+              <juju.components.SvgIcon
+                height="30"
+                name="juju-logo"
```

Comment:
You could target a `className` with the styles here in case someone adds another icon in this component.